### ISSUE TITLE
Provide option to increase pin spacing with pin_order.cfg

### ISSUE THREAD

**Issue by vijayank88** [The-OpenROAD-Project/OpenLane#1711]
### Description

Based on comment https://github.com/The-OpenROAD-Project/OpenLane/issues/1699#issuecomment-1488316946
For IO limited block with fixed pin location through `pin_order.cfg` it is good to add `FP_IO_MIN_DISTANCE` configuration to keep some minimum distance.

![pin_order_issue](https://user-images.githubusercontent.com/85787913/229271645-9b6643ca-83fb-49e6-8894-346886396c72.png)
in above floorplan, the corner has some more space. As existing pin placement not increasing pin distance. It is good to customize pin distance for better pin placement.

So it will be easy to place buffer and routing, otherwise facing congestion issue.

**Comment by kareefardi** [comment]
@vijayank88 what is the exact configuration that you used to take that screenshot?

**Comment by vijayank88** [comment]
Increased die size 1600x3010 and ran the OpenLane flow with user design of fixed pin_order.cfg.

**Comment by vijayank88** [comment]
Yes, I agree. Die size should be increased to place all the pins...

**Comment by hanm2019** [comment]
make be place in multiple layer? Can I decide the pin's layer also in pin_order.cfg? such as
```
#LAYER 1
#N 
clk
rst
bypass

#S
xxxx

#LAYER 2
#N 
...
...

#S
...
...

```
